SPECS/systemd/systemd.spec

```diff
@@ -856,6 +857,8 @@ install -Dm0644 10-timeout-abort.conf.user %{buildroot}%{user_unit_dir}/service.
 # https://fedoraproject.org/wiki/Changes/IncreaseVmMaxMapCount
 install -Dm0644 -t %{buildroot}%{_prefix}/lib/sysctl.d/ %{SOURCE17}
 
+install -Dm0644 -t %{buildroot}%{_prefix}/lib/sysctl.d/ %{SOURCE18}
```

Comment:
While we haven't universally done this, I think it would be nice to add at least a comment about this being unique to `azl`, or one of:
```spec
%if 0%{?azl}
install -Dm0644 -t %{buildroot}%{_prefix}/lib/sysctl.d/ %{SOURCE18}
%endif
```
``` spec
%{?azl:install -Dm0644 -t %{buildroot}%{_prefix}/lib/sysctl.d/ %{SOURCE18}}
```

Reply:
added macros :)